documentation/modules/rn-2.2.adoc

```diff
@@ -94,3 +94,37 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Restart of the CatalogSource pod is required to upgrade MTV 2.1.0 to MTV 2.2.0.
+
+To restart the CatalogSource pod, do the following:
```

Comment:
Pod names are marked up in backticks.

You need to indicate at what point this needs to be done. Is this before the upgrade? If so, state it.

Explain why the pod needs to be restarted. Something like "You must restart the `CatalogSource` pod before upgrading ... because ...."